TLM/TLM/TrafficManagerMod.cs

```diff
@@ -53,7 +53,9 @@ public class TrafficManagerMod : IUserMod {
 
         internal bool InGameHotReload { get; set; } = false;
 
-        internal static bool InGame() => SceneManager.GetActiveScene().name == "Game";
+        internal static bool InGame() =>
```

Comment:
Add `///` doc comment explaining what this `InGame` means, does it include editor mode?